### ISSUE TITLE
I could not make github oath tokens by bloom

### ISSUE THREAD

**Issue by hakuturu583** [ros-infrastructure/bloom#611]
I tried to release embree_vendor package for Foxy. (https://github.com/OUXT-Polaris/embree_vendor)
Release process works well, but I could not make github oath tokens in bloom-release.
So, I could not release package.
I manually create github oath tokens and add it to ~/.config/bloom, then I successfully release my package.

**Comment by hakuturu583** [comment]
command line output when I failed to release my package is here.

```
Looks like bloom doesn't have an oauth token for you yet.
Therefore bloom will require your GitHub username and password just this once.
With your GitHub username and password bloom will create an oauth token on your behalf.
The token will be stored in `~/.config/bloom`.
You can delete the token from that file to have a new token generated.
Guard this token like a password, because it allows someone/something to act on your behalf.
If you need to unauthorize it, remove it from the 'Applications' menu in your GitHub account page.

Would you like to create an OAuth token now [Y/n]? Y
GitHub username [masaya]: hakuturu583
GitHub password (never stored): 
HTTP Error 404: Not Found (https://api.github.com/authorizations): None
```

**Comment by Levi-Armstrong** [comment]
I recently ran into this issue. To resolve the issue got to you your github developers setting and create a personal access token and give access to repo. Then create the following file `~/.config/bloom` on your machine and add the following content below.

```
{ "github_user": "Add github username", "oauth_token": "Add Generated Token Here"}
```

**Comment by kmhallen** [comment]
Bloom is still unable to make github oath tokens with version `0.10.3`. Manually creating a token works.
The `workflow` token permission is needed to create a pull request now. This must be a recent github change.

**Comment by clalancette** [comment]
> The `workflow` token permission is needed to create a pull request now. This must be a recent github change.

We've run into this elsewhere.  This is caused by the fact that Github workflows did change in rosdistro, so trying to fast-forward your branch "through" them requires the token.  The way around this (if you don't want to give your token `workflow` permission) is to manually update the master branch on your fork to the latest.  That should remove the requirement.

**Comment by fmessmer** [comment]
is this related to GitHub’s new authentication token formats
https://github.blog/2021-04-05-behind-githubs-new-authentication-token-formats/

**Comment by nuclearsandwich** [comment]
@fmessmer those tokens create new problems which will be addressed by #627.

I've also got #628 open which updates the text for first time setup to explain (minimally) why the workflow scope is needed. Feedback on either or both is welcome.

**Comment by nuclearsandwich** [comment]
I'm going to close this issue since Bloom will no longer attempt to create access tokens on your behalf but as of 0.10.4 will now point you to the url where you can create a token, come back and drop it in.

Thanks for reporting the issues with token creation.
